src/rules/MD/MDRules.js

```diff
@@ -0,0 +1,122 @@
+const util = require("../../util");
+const fs = require("fs");
+const os = require('os');
+
+module.exports = {
+  /**
+   * Analyzes whether the first section of the file is of the determined order.
+   * 
+   * @param {*} link File path.
+   * @param {*} rules Rules file.
+   */
+  ruleFirstSectionStartsWithHx: function (link, rules) {
+    let result = {
+      status: '',
+      line: '-',
+      column: '-',
+      msg: '',
```

Comment:
What's the difference between `msg` and `toString`? If there's a significant difference, could we change the name to `message`? I'm curious if we really need of it